handler/src/main/java/io/netty/handler/ssl/OpenSslServerContext.java

```diff
@@ -346,65 +350,27 @@ private OpenSslServerContext(
         // Create a new SSL_CTX and configure it.
         boolean success = false;
         try {
-            checkKeyManagerFactory(keyManagerFactory);
             checkNotNull(keyCertChain, "keyCertChainFile");
             checkNotNull(key, "keyFile");
 
-            if (keyPassword == null) {
-                keyPassword = "";
-            }
-
+            keyPassword = keyPassword == null ? StringUtil.EMPTY_STRING : keyPassword;
```

Comment:
@normanmaurer - sgtm ... the `OpenSslClientContext` should also be investigated.

 at the very least it seems redundant though ... should `OpenSslKeyMaterialManager` and `setCertificateBio` allow for a `null` password? `setCertificateBio` may have to be updated to null out `cb_data->password[0]` ... not sure